### PR TITLE
SW-1339 Remove unused fields from seed summary API

### PR DESCRIPTION
**Pull request by sgrimm**
To support simplifying the accessions dashboard, remove some values from the
summary API that we're no longer going to be showing and thus don't need to spend
time calculating:

* The number of families. We're now only showing species counts.
* The "last week" statistics for active accessions and species. We're not going to
  show percentage changes any more.

The latter change isn't backward-compatible; it lifts the value from a child
object up into the top-level `SummaryResponse` payload.

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #527** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/527" target="_blank">Graphite</a>  👈
    * **PR #528** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/528" target="_blank">Graphite</a> 
      * **PR #529** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/529" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/527?utm_source=stack-comment).

**Comment by sgrimm** [comment]
@karthikbtf We shouldn't merge this until after the FE is updated to cope with both the old and new payload structures. Putting this PR up now so you can pull it and test your FE changes against it.

**Comment by sgrimm** [comment]
Changed it to a draft PR so we don't merge it by accident, but the code is ready for review.

**Review by karthikbtf — APPROVED** [review]
Looks good to me.

**Comment by karthikbtf** [comment]
> Changed it to a draft PR so we don't merge it by accident, but the code is ready for review.

This should be ready to merge. We will need a followup review in FE after that, copying @constanzauanini for visibility.

**Comment by sgrimm** [comment]
**Graphite Merge Job**

Current status: ✅ Merged

This pull request was successfully merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/527?utm_source=stack-comment).


Job Reference: I5iKiK2k4z2BqX6fGLr8

**Pull request merged**
